modules/integration/tests-common/clients/admin/src/gen/java/org/wso2/am/integration/clients/admin/api/ImportExportApi.java

```diff
@@ -0,0 +1,334 @@
+ *
+ * The version of the OpenAPI document: v2
+ * Contact: architecture@wso2.com
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+package org.wso2.am.integration.clients.admin.api;
+
+import org.wso2.am.integration.clients.admin.ApiCallback;
+import org.wso2.am.integration.clients.admin.ApiClient;
+import org.wso2.am.integration.clients.admin.ApiException;
+import org.wso2.am.integration.clients.admin.ApiResponse;
+import org.wso2.am.integration.clients.admin.Configuration;
+import org.wso2.am.integration.clients.admin.Pair;
+import org.wso2.am.integration.clients.admin.ProgressRequestBody;
+import org.wso2.am.integration.clients.admin.ProgressResponseBody;
+
+import com.google.gson.reflect.TypeToken;
+
+import java.io.IOException;
+
+
+import org.wso2.am.integration.clients.admin.api.dto.ErrorDTO;
+import org.wso2.am.integration.clients.admin.api.dto.ExportThrottlePolicyDTO;
+import java.io.File;
+
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ImportExportApi {
```

Comment:
These generated Class names seems not intuitive. Should convene that these are related to "Throttling Policies". Check how these can be changed.

Reply:
Fixed with 3106d801961d2d8e82a5a35117427b8337a92455